autoscale/src/main/java/com/sequenceiq/periscope/domain/ScalingTrigger.java

```diff
@@ -0,0 +1,118 @@
+package com.sequenceiq.periscope.domain;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.sequenceiq.periscope.api.model.TriggerStatus;
+
+@Entity
+@Table(name = "scaling_trigger")
+public class ScalingTrigger implements Clustered {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "scaling_trigger_generator")
+    @SequenceGenerator(name  = "scaling_trigger_generator", sequenceName = "scaling_trigger_id_seq", allocationSize = 1)
+    private Long id;
+
+    @Column(name = "trigger_crn")
+    private String triggerCrn;
+
+    @Column(name = "flow_id")
+    private String flowId;
+
+    @Column(name = "start_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startTime;
+
+    @Column(name = "yarn_metrics")
```

Comment:
This column could potentially be re-purposed to include additional information like the reason for a trigger to fail. (403 responses, YARN errors, etc)

Reply:
Renamed this column